cmd/analyzer/analyzer.go

```diff
@@ -136,10 +138,59 @@ func wipeStorage(cfg *config.StorageConfig) error {
 type Service struct {
 	Analyzers map[string]analyzer.Analyzer
 
+	source *AllSources
 	target storage.TargetStorage
 	logger *log.Logger
 }
 
+type AllSources struct {
```

Comment:
AbstractServiceClientFactoryAggregator here we come

Reply:
yeah... I was also considering `SourceFactory` but it's not quite satisfactory either